e2e/assets/status/chart-with-template-vars/.helmignore

```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
+# This supports shell glob matching, relative path matching, and
+# negation (prefixed with !). Only one pattern per line.
+.DS_Store
+# Common VCS dirs
+.git/
+.gitignore
+.bzr/
```

Comment:
Do we really need all these patterns in this case, or even this file? :upside_down_face:

Reply:
You're right. I don't think we need that file at all.